third_party/move/move-model/src/model.rs

```diff
@@ -3015,12 +3013,12 @@ impl<'env> ModuleEnv<'env> {
     }
 
     /// Returns true if functions in the current module can call a public(package) function in the given module.
```

Comment:
In this PR: https://github.com/aptos-labs/aptos-core/pull/13680/files#diff-d91ccc31f071f5356fd31e9d48dc8372b55377e257f4f2d2a1fdebdafa78a48f, in `function_checker.rs`,  `is_target()` was changed to `is_primary_target()`.
1. Among those, for the visibility check cases, can you add a similar TODO (so that once we properly add friend decls for non-primary targets for `public(package)`, we perform visibility checks on dependencies' sources as well, which is the v1 behavior).
2. For the non-visibility-check cases, can you double check if the change from `is_target()` to `is_primary_target()` was valid, and briefly mention why, as a reply here?